opentelekomcloud/services/as/resource_opentelekomcloud_as_configuration_v1.go

```diff
@@ -65,9 +65,10 @@ func ResourceASConfiguration() *schema.Resource {
 							Required: true,
 						},
 						"user_data": {
-							Type:     schema.TypeString,
-							Optional: true,
-							ForceNew: true,
+							Type:             schema.TypeString,
+							Optional:         true,
+							ForceNew:         true,
+							DiffSuppressFunc: common.SuppressEmptyLineSHA,
```

Comment:
It's more like "empty string", than "empty line"

Reply:
done